server/src/weaverbird/backends/pypika_translator/translators/base.py

```diff
@@ -797,8 +801,8 @@ def _domain(self: Self, *, step: "DomainStep") -> StepContext:
                 raise NotImplementedError(f"[{self.DIALECT}] Cannot resolve a reference to a query")
             else:
                 selected_cols = list(self._tables_columns[step.domain])
-        except KeyError:
-            selected_cols = ["*"]
+        except KeyError as exc:
+            raise UnknownTableColumns(f"Table {exc} not in table_columns") from exc
```

Comment:
```suggestion
        except KeyError as asked_table:
            raise UnknownTableColumns(f"Table {asked_table!r} not in table_columns") from exc
```

Reply:
Je comprends pas le `!r` :thinking: Ça nous donnerait `Table KeyError('my table') not in table_columns`

Reply:
ah oups lol 🙈 
je pensais `except KeyError as exc` puis `{exc.args[0]!r}` pour récupérer que la clé
Mais laisse comme ca si tu veux pas de souci !

Reply:
Oui je préfère comme ça, comme ça on a déja les quotes :smile: